src/bitmessagekivy/kv/popup.kv

```diff
@@ -29,7 +29,7 @@
             required: True
             icon_right: 'label'
             helper_text_mode: "on_error"
-            on_text: root.checkLabel_valid(self)
+            # on_text: root.checkLabel_valid(self)
```

Comment:
make it into a TODO instead of a comment

Reply:
Done with it